metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/EnrichmentWorkerTest.java

```diff
@@ -35,10 +39,6 @@
 import eu.europeana.enrichment.utils.EntityMergeEngine;
 import eu.europeana.enrichment.utils.InputValue;
 
-// TODO Tests that could be added:
-// 1. What if there is nothing to process (empty or null lists are returned when extracting or when
-// enriching/dereferencing).
-// 2. Throwing exceptions
 @RunWith(MockitoJUnitRunner.class)
```

Comment:
This class is deprecated. Maybe switch to the alternative suggested in that class's javadoc.